src/wasm/WasmModule.cpp

```diff
@@ -60,43 +76,58 @@ wasm::WasmEnvironment& WasmModule::getWasmEnvironment()
     return wasmEnvironment;
 }
 
-std::string WasmModule::snapshot()
+std::string WasmModule::snapshot(bool locallyRestorable)
```

Comment:
This `locallyRestorable` parameter is passed through to Faabric to signify whether a snapshot is intended purely for remote process/ thread spawning